metrics/report/README.md

```diff
@@ -27,6 +27,10 @@ Repeat this process if you want to compare multiple sets of results. Note, the
 report generation scripts process all subfolders of `tests/metrics/results` when
 generating the report.
 
+> **Note:** By default, the `grabdata.sh` script ties to launch some moderately
+> large containers (8Gbyte RAM), so may fail to produce some results on a memory
```

Comment:
large containers (i.e. 8Gbyte RAM) and might fail to produce some results on a memory ...